packages/config/src/contracts/moonbeam/index.ts

```diff
@@ -0,0 +1,6 @@
+import xtokens from "./xtokens";
+
+const moonbeam = { xtokens };
+
+// eslint-disable-next-line import/no-default-export
+export default moonbeam;
```

Comment:
`export { moonbeam }`; since we will add more chains to this object later, such as `{acala, moonbeam, astar}`

Reply:
Fixed